guides/main_concepts/documents.md

```diff
@@ -155,13 +155,13 @@ Take note that the document addition request in MeiliSearch is <!-- prettier-ign
 
 By default, MeiliSearch limits the size of `JSON` payloads—and therefore document uploads—to 10MB.
 
-To upload more documents in one go, it is possible to [change the payload size limit](/guides/advanced_guides/configuration.md#payload-limit-size) during the setup of your MeiliSearch instance using the `http-payload-size-limit` option.
+To upload more documents in one go, it is possible to [change the payload size limit](/guides/advanced_guides/configuration.md#payload-limit-size) during the setup of your MeiliSearch instance using the `http-payload-size-limit` option. The new limit must be given in bytes.
 
 ```bash
-$ ./meilisearch --http-payload-size-limit=100000000
+$ ./meilisearch --http-payload-size-limit=1000000000
 ```
 
-> The payload limit is now +-100MB instead of 10MB
+> The above code sets the payload limit to approximately 1GB, instead of the 10MB default.
```

Comment:
This is correct.

For it to be compatible with the next release, It has to change with the following : 
> The above code sets the payload limit to approximately 1GB, instead of the 100MB default.

In this PR:  https://github.com/meilisearch/documentation/pull/673